lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -90,6 +113,18 @@ def check_ndjson(ndjson, type)
         end
       end
 
+      def check_delete_request(url)
+        reply = @client.delete(url, {})
+        skip 'Server did not accept client request to delete export file after export completed' if reply.code > 400
```

Comment:
Is the skip really needed? The test is marked `optional`.

Reply:
Yes since "not accept client delete request" is also a valid response. This test is focus on the return code after server accepted client's request. Test 08 is already marked as optional. Will mark Test 09 as optional too.